### PR TITLE
searchneu now shows course attributes on each search!

### PR DESCRIPTION
**Pull request by NEUDitao**
https://github.com/ryanhugh/searchneu/issues/14

There's a collapsible box now. You click on it and it's beautiful.

### CODE REVIEW

**Comment by NEUDitao** [comment]
i'm dumb it's not ready yet U_U

**Comment by NEUDitao** [comment]
now we're ready!

File: backend/scrapers/classes/parsers/baseParser.js [hunk 1]
```diff
@@ -219,6 +219,19 @@ class BaseParser {
...
+  // gets the course Attributes. If course attributes don't exist, return null
+  // String -> [Maybe String]
+  parseStacysMom(gotItGoingOn) {
+    let courseAttributes = gotItGoingOn.match(/(Course Attributes: \n)(.*)/i);
+    if (courseAttributes) {
+      courseAttributes = courseAttributes[2].trim().split(', ');
+      for (let i = 0; i < courseAttributes.length; i++) {
+        courseAttributes[i] = courseAttributes[i].trim();
+      }
+    }
```
Comment (ryanhugh) [inline backend/scrapers/classes/parsers/baseParser.js]: want to do a 
```
else {
return null;
}
```
?

Just so its clear what we're returning if the .match didn't match anything

File: backend/scrapers/classes/parsers/ellucianCatalogParser.js [hunk 2]
```diff
@@ -79,7 +79,15 @@ class EllucianCatalogParser extends EllucianBaseParser.EllucianBaseParser {
...
-      macros.log('warning, nothing matchied credits', url, text);
+      macros.log('warning, nothing matched credits', url, text);
+    }
+
+    const courseAttributes = this.parseStacysMom(text);
+
+    if (courseAttributes) {
+      depData.courseAttributes = courseAttributes;
+    } else {
+      macros.log('warning, nothing matched course attributes', url, text);
```
Comment (ryanhugh) [inline backend/scrapers/classes/parsers/ellucianCatalogParser.js]: Also lets remove logging of the text here, its quite a lot of output when running this code

**Comment by NEUDitao** [comment]
did the things

**Comment by NEUDitao** [comment]
uno

**Pull request merged**